### PR TITLE
services/hound: remove

### PR DESCRIPTION
**Pull request by zowoq**
https://docs.github.com/en/search-github/github-code-search/about-github-code-search

Will wait until the new code search isn't in beta.

### CODE REVIEW

**Comment by Mic92** [comment]
Sounds good in principle

**Review by zimbatm — APPROVED** [review]
sounds good!

**Comment by zowoq** [comment]
https://github.blog/changelog/2023-05-08-the-new-code-search-and-code-view-is-now-generally-available/

Doesn't actually seem to be GA for everyone yet.

**Comment by zimbatm** [comment]
I doubt many people even knew that hound existed

File: terraform/cloudflare_nix-community_org.tf [hunk 1]
```diff
@@ -102,10 +102,21 @@ resource "cloudflare_record" "nix-community-org-nur-update-CNAME" {
 resource "cloudflare_record" "nix-community-org-search-CNAME" {
   zone_id = local.nix_community_zone_id
   name    = "search"
-  value   = "build03.nix-community.org"
+  value   = "nix-community.org"
   type    = "CNAME"
 }
 
+resource "cloudflare_page_rule" "nix-community-org-search-forward" {
```
Comment (zowoq) [inline terraform/cloudflare_nix-community_org.tf:109]:
```
Error: failed to create page rule: Unauthorized to access requested resource (9109)
```

Comment (zowoq) [reply terraform/cloudflare_nix-community_org.tf:109]: Can create it manually but it doesn't do anything.

File: terraform/cloudflare_nix-community_org.tf [hunk 2]
```diff
@@ -102,9 +102,23 @@ resource "cloudflare_record" "nix-community-org-nur-update-CNAME" {
...
-  value   = "build03.nix-community.org"
+  value   = "nix-community.org"
   type    = "CNAME"
-}
+  proxied = true
+}
+
+# created manually
+#
+#resource "cloudflare_page_rule" "nix-community-org-search-forward" {
```
Comment (zowoq) [inline terraform/cloudflare_nix-community_org.tf:112]: I'll leave this as is for now and delete it in a couple of months.

**Pull request merged**